frontend/packages/kubevirt-plugin/src/selectors/vm-like/nextRunChanges.scss

```diff
@@ -0,0 +1,4 @@
+.kubevirt-vm-details__restart_required-class-alert {
```

Comment:
here too, we have plenty of margin classes in `_common.scss`

Reply:
I can't find there something that fits. I'll leave it.